cdap-ui/app/cdap/components/PipelineList/DeployedPipelineView/PipelineTable/index.tsx

```diff
@@ -110,13 +147,10 @@ const PipelineTableView: React.SFC<IProps> = ({
           <div className="grid-row">
             <SortableHeader columnName="name" />
             <SortableHeader columnName="type" />
-            <SortableHeader columnName="status" disabled={Object.keys(statusMap).length === 0} />
-            <SortableHeader
-              columnName="lastStartTime"
-              disabled={Object.keys(statusMap).length === 0}
-            />
+            <SortableHeader columnName="status" disabled={pipelines.length === 0} />
+            <SortableHeader columnName="lastStartTime" disabled={pipelines.length === 0} />
```

Comment:
the disabled state should be able to be removed. This was added because the fetching of the status and nextRunTimes were being done async.

Reply:
Done